bindings/python/iota_sdk/utils.py

```diff
@@ -172,7 +172,7 @@ def compute_token_id(account_id: HexStr, serial_number: int,
         })
 
     @staticmethod
-    def block_id(block: Block) -> HexStr:
+    def block_id(block: BlockWrapper) -> HexStr:
```

Comment:
so this needs protocolparams added. not surewhat to do with the .id() method. Maybe also just add the param? 🤢